TrivialSerialisation/Common/interface/TrivialSerialiser.h

```diff
@@ -0,0 +1,144 @@
+#ifndef TrivialSerialisation_Common_TrivialSerialiser_h
+#define TrivialSerialisation_Common_TrivialSerialiser_h
+
+#include <cstdio>
+#include <vector>
+#include <span>
+#include <cstddef>
+#include <type_traits>
+
+#include "DataFormats/Common/interface/Wrapper.h"
+#include "DataFormats/Common/interface/AnyBuffer.h"
+#include "FWCore/Utilities/interface/EDMException.h"
+#include "FWCore/Utilities/interface/TypeDemangler.h"
+#include "TrivialSerialisation/Common/interface/TrivialSerialiserBase.h"
+#include "DataFormats/Common/interface/TrivialCopyTraits.h"
+
+// defines all methods of TrivialSerialiserBase
+
+namespace ngt {
+
+  template <typename T>
+  class TrivialSerialiser : public TrivialSerialiserBase {
+  public:
+    using WrapperType = edm::Wrapper<T>;
+    TrivialSerialiser(WrapperType const& obj_) : TrivialSerialiserBase(&obj_) {}
```

Comment:
```suggestion
    TrivialSerialiser(WrapperType const& obj) : TrivialSerialiserBase(&obj) {}
```

Names with a trailing underscore should be used (only) for data members.